open_dread_rando/game_patches.py

```diff
@@ -41,10 +41,14 @@ def apply_game_patches(editor: PatcherEditor, configuration: dict):
     if configuration["remove_grapple_blocks_hanubia_shortcut"]:
         for reference in _HANUBIA_SHORTCUT_GRAPPLE_BLOCKS:
             editor.remove_entity(reference)
+            bmmap = editor.get_scenario_file(reference["scenario"], Bmmap)
+            bmmap.raw.Root.mapProps.pop(reference["actor"])
 
```

Comment:
I'm gonna guess this is gonna be repeated behavior, might as well put it in the `remove_entity` function imo, e.g.: `editor.remove_entity(reference, map_category="mapProps")`